src/tools/kata-ctl/src/main.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Result};
+use clap::{crate_name, crate_version, App, Arg, SubCommand};
+use std::process::exit;
+
+mod utils;
+
+const DESCRIPTION_TEXT: &str = r#"DESCRIPTION:
+    kata-ctl description placeholder."#;
+
+const ABOUT_TEXT: &str = "Kata Containers kata control tool";
+
+fn real_main() -> Result<()> {
+    let name = crate_name!();
+
+    let app = App::new(name)
+        .version(crate_version!())
+        .about(ABOUT_TEXT)
+        .long_about(DESCRIPTION_TEXT)
+        .subcommand(
+            SubCommand::with_name("version")
+            .about("display version details")
+        )
+        .subcommand(
+            SubCommand::with_name("kata-check")
+            .about("tests if system can run Kata Containers")
+        )
+        .subcommand(
+            SubCommand::with_name("kata-env")
```

Comment:
I think we can drop the prefixes as those were only required back in the day when we wanted to avoid any possible clash with `runc` args.

```suggestion
            SubCommand::with_name("check")
            .about("tests if system can run Kata Containers")
        )
        .subcommand(
            SubCommand::with_name("env")
```